thirdeye/thirdeye-frontend/app/pods/components/te-navbar/template.hbs

```diff
@@ -0,0 +1,40 @@
+<div class="container">
+  {{#if isGlobalNav}}
+    <div class="te-nav__logo pull-left">
+      {{#link-to "index" class="te-nav__link te-nav__link--home"}}
+        {{webappName}}
+      {{/link-to}}
+    </div>
+  {{/if}}
+
+  <ul class="te-nav__list pull-left {{navClass}}">
+    {{#each navItems as |navItem|}}
+      <li class="te-nav__item te-nav__item--{{navItem.className}}">
+        {{#if navItem.isCustomLink}}
+          <a class="te-nav__link nounderline" href="{{navItem.link}}">
+            {{navItem.title}}
+          </a>
+        {{else if navItem.id}}
+          {{#link-to navItem.link navItem.id class="te-nav__link app-text--xxsmall"}}
```

Comment:
is `app-text--xxsmall` defined anywhere? 
nit: instead of global classes, I would rather have this as a mixins that we can include in `.te-nav__link`